### PR TITLE
Alert actions

### PR DESCRIPTION
**Pull request by arturdryomov**
There is no actual alert removing due to API reasons.

device-2015-06-19-205240

### CODE REVIEW

**Comment by arturdryomov** [comment]
Rebased.

**Comment by arturdryomov** [comment]
Rebased once again.

File: src/main/java/org/hawkular/client/android/activity/AlertsActivity.java [hunk 1]
```diff
@@ -75,11 +77,50 @@ private void showProgress() {
...
+        alertMenu.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(MenuItem menuItem) {
+                switch (menuItem.getItemId()) {
+                    case R.id.menu_resolve:
+                        removeAlert(alertPosition);
+                        return true;
+
+                    case R.id.menu_acknowledge:
+                        removeAlert(alertPosition);
```
Comment (pilhuhn) [inline src/main/java/org/hawkular/client/android/activity/AlertsActivity.java]: Looks odd - shouldn't that different actions?

Comment (arturdryomov) [inline src/main/java/org/hawkular/client/android/activity/AlertsActivity.java]: The rebase process messed things up a little. As I mentioned at the description there is no actual action processing at this point due to [API conflicts](http://lists.jboss.org/pipermail/hawkular-dev/2015-June/001191.html).

**Comment by arturdryomov** [comment]
And again!

**Pull request merged**